tests/search/includes/classes/test-class-queue.php

```diff
@@ -1228,14 +1235,7 @@ function() {
 			}
 		);
 
-		$this->expectNotice();
-		$this->expectNoticeMessage(
-			sprintf(
-				'add_filter was called <strong>incorrectly</strong>. %s should be an integer. Please see <a href="https://wordpress.org/support/article/debugging-in-wordpress/">Debugging in WordPress</a> for more information. (This message was added in version 5.5.3.)',
-				$filter
-			)
-		);
-
+		$this->setExpectedIncorrectUsage( 'add_filter' );
```

Comment:
Now that we invoke `parent::setUp()`, we have to use the "canonical" way of catching `doing_it_wrong()`s